run--03b--postestimation-disaggregate-estimates.Rmd

```diff
@@ -350,7 +350,7 @@ add_agg_level <- function(ests, agg_level) {
         geo_crosswalk %>% 
         transmute(GEOID    = GEOID, 
                   County   = County, 
-                  geo_part = pct_area) %>% 
+                  geo_part = 1) %>% 
```

Comment:
Not sure why this has to be changed to 1 -- I know we need to fix the pct_area for county. Is this a temporary fix?

Reply:
It's at least a temporary fix, yes. But: isn't it the case that tracts tend to line up perfectly with county boundaries?

Reply:
For just the county, yes. But when we incorporate the aux geo (CCA) I found an issue when separating the county into Chicago vs non-Chicago (there's an email thread with Bryan while you were on vacation). We should revisit that issue later.